src/api/university/client/useGetMyWishUniversity.ts

```
@@ -0,0 +1,25 @@
+export const getMyWishUniversity = (): Promise<AxiosResponse<ListUniversity[]>> =>
+  axiosInstance.get("/univ-apply-infos/like");
+
+const useGetMyWishUniversity = () => {
+  return useQuery({
+    queryKey: [QueryKeys.univApplyInfosLike],
+    queryFn: () => getMyWishUniversity(),
+    staleTime: 1000 * 60 * 5,
+    select: (data) => data.data,
+  });
+
+  return;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)